packages/typescript-versions/src/index.ts

```diff
@@ -52,20 +52,21 @@ export type UnsupportedTypeScriptVersion =
   | "4.0"
   | "4.1"
   | "4.2"
-  | "4.3";
+  | "4.3"
+  | "4.4";
 /**
  * Parseable and supported TypeScript versions.
  * Only add to this list if we will support this version on Definitely Typed.
  */
-export type TypeScriptVersion = "4.4" | "4.5" | "4.6" | "4.7" | "4.8" | "4.9" | "5.0" | "5.1" | "5.2";
+export type TypeScriptVersion = "4.5" | "4.6" | "4.7" | "4.8" | "4.9" | "5.0" | "5.1" | "5.2" | "5.3";
```

Comment:
according to the RC directions at the top of the file, you should only add 5.3, not remove 4.4.